src/app/components/pool/graphs/pool-graphs.html

```diff
@@ -1,9 +1,11 @@
 <bl-banner *ngIf="startTaskFailedError" fixMessage="Edit the start task" [fix]="openEditStartTask">
-    <div message>It seems like the start task is failing on many nodes.
-        ({{startTaskFailedError.failedNodes}}/{{startTaskFailedError.nodeCount}})</div>
+    <div message>It seems like the start task is failing on {{startTaskFailedError.failedStartTaskNodes}} out of {{startTaskFailedError.nodeCount}} nodes.</div>
     <div [other-fix]="rebootFailedNodes" fixMessage="Reboot failed nodes"></div>
     <div *ngIf="isPaasPool" [other-fix]="reimageFailedNodes" fixMessage="Reimage failed nodes"></div>
 </bl-banner>
+<bl-banner *ngIf="unusableNodeError">
+    <div message>It seems like {{unusableNodeError.failedUnusableNodes}} out of {{unusableNodeError.nodeCount}} nodes are unusable.</div>
```

Comment:
```suggestion
    <div message>{{unusableNodeError.failedUnusableNodes}} out of {{unusableNodeError.nodeCount}} nodes are in an unusable state.</div>
```

Maybe also add some language describing how to investigate further ie hover or click into node